src/main/resources/templates/clientcontact/list.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="cs"
+      xmlns:th="http://www.springframework.org/schema/data/jaxb">
```

Comment:
Ten xmlns by myslim obsahuje jinou adresu. Nekde byla thymelead url. Ale uz nevim v ktere sablone to je dobre.

Reply:
<html lang="cs" xmlns:th="http://www.thymeleaf.org">    je tohle spravne?

Reply:
je tohle spravne?    <html lang="cs" xmlns:th="http://www.thymeleaf.org">

Reply:
html lang="cs" xmlns:th="http://www.thymeleaf.org"

Reply:
jo, to by  melo byt ono :)

Reply:
upraveno